leader-election-impl/src/main/java/com/palantir/leader/LeadershipEvents.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.leader;
+
+import com.codahale.metrics.Meter;
+import com.palantir.atlasdb.util.AtlasDbMetrics;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.paxos.PaxosRoundFailureException;
+import com.palantir.paxos.PaxosValue;
+
+public class LeadershipEvents {
+
+    private final Meter gainedLeadership = AtlasDbMetrics.getMetricRegistry().meter("leadership.gained");
+    private final Meter lostLeadership = AtlasDbMetrics.getMetricRegistry().meter("leadership.lost");
+    private final Meter noQuorum = AtlasDbMetrics.getMetricRegistry().meter("leadership.no-quorum");
+    private final Meter proposedLeadership = AtlasDbMetrics.getMetricRegistry().meter("leadership.proposed");
+    private final Meter proposalFailure = AtlasDbMetrics.getMetricRegistry().meter("leadership.proposed.failure");
```

Comment:
Nit: might want to take MetricRegistry as constructor or factory arg so we're not tightly coupled to the global metric registry for testing